package.json

```diff
@@ -13,7 +13,7 @@
     "url": "https://github.com/hubot-scripts/hubot-redis-brain/issues"
   },
   "dependencies": {
-    "redis": "0.8.4"
+    "redis": "~0.12.x"
```

Comment:
I think I'd rather see a specific version locked down here, so it's easier to be using a consistent version across development vs production.